docs/src/userguide/glossary.rst

```diff
@@ -0,0 +1,194 @@
+.. _glossary:
+
+Glossary
+=============
+
+.. glossary::
+
+   Cartopy
+        A python package for producing maps, and other geospatial data.
+        Allows plotting on these maps, over a range of projections.
+
+        | **Related:** :term:`Matplotlib`
+        | **More information:** `CartoPy Site <https://scitools.org.uk/cartopy/docs/latest/>`_
+        |
+
+   Coordinates
+        A container for data points, comes in three main flavours.
+
+        - Dimensional Coordinates -
+            A coordinate that represents a singular data dimension.
+            These are organised in ascending order of dimension. Can only
+            contain numerical data.
+        - Auxiliary Coordinates -
+            A coordinate that can map to multiple data dimensions. Can
+            contain any type of data.
+        - Scalar Coordinates -
+                A coordinate that is not mapped to any data dimension, instead
+                representing the cube as a whole.
+
+        | **Related:** :term:`Cubes`
+        | **More information:** :doc:`iris_cubes`
+        |
+
+   Cubes
+        Cubes are the main method of storing data in Iris. A cube can consist of:
+
+        - Array of :term:`Phenomenon` Data (Required)
+        - :term:`Coordinates`
+        - :term:`Standard Name`
+        - :term:`Long Name`
+        - :term:`Units`
+        - :term:`Cell Method`
```

Comment:
```suggestion
        - :term:`Cell Methods`
```

Reply:
Was unsure on this one; due the nature of links, I have to change the heading of the linked terms as well (Cell Method glossary item is now called Cell Methods also). Have changed it now, was very much on the fence and think you've swayed me.

Reply:
I see the problem + sympathize.
But FWIW I'd choose not to have glossary terms pluralised.
However, I **_think_** you can fix this by providing a custom link text : https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-syntax
I'm just going to try that one out ...